checkov/common/checks_infra/solvers/connections_solvers/and_connection_solver.py

```diff
@@ -35,10 +35,10 @@ def get_operation(self, graph_connector: LibraryGraph) -> \
             unknown.extend(unknown_solver)
             failed_or_unknown_ids.extend(f[CustomAttributes.ID] for f in itertools.chain(failed_solver, unknown_solver))
```

Comment:
do we want to add it here as well? or delete this line as we saw it might be not relevant